pkg/drivers/kic/kic.go

```diff
@@ -130,7 +130,7 @@ func (d *Driver) Create() error {
 			return
 		}
 		t := time.Now()
-		glog.Infof("Starting extracting preloaded images to volume")
+		glog.Infof("Starting extracting preloaded images to volume ....")
```

Comment:
nit: we typically use three dots for logs

Reply:
done